humanguide/other-design-considerations/accessibility.rst

```diff
@@ -0,0 +1,17 @@
+Accessibility
+=============
+
+Accessibility is important to make all users to be able to use the operating system.
+
+Inclusive palette and theme
+---------------------------
+
+We would like to create and implement color palettes helpful to people who have sight impairment:
+- Additional `palette proposal <http://colors.mivoligo.com/#color-blind>`__.
+- A high contrast theme.
+- Adaptable font size theme.
```

Comment:
Sorry, I think only asterisks (`*`) can be used to make a bulleted list. Or, maybe there needs to be a blank line above the bulleted list.